config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@everybody I think @jodh-intel proposal is the best, that is take `/usr/local/bin/qemu-q35-system-x86_64` as the path for Q35. And eventually we will move to `qemu-system-x86_64` when everything will be merged to upstream.